lib/delivery_boy/instance.rb

```diff
@@ -95,7 +95,8 @@ def kafka
         sasl_scram_username: config.sasl_scram_username,
         sasl_scram_password: config.sasl_scram_password,
         sasl_scram_mechanism: config.sasl_scram_mechanism,
-        sasl_over_ssl: config.sasl_over_ssl
+        sasl_over_ssl: config.sasl_over_ssl,
+        sasl_oauth_token_provider: config.sasl_oauth_token_provider
```

Comment:
@dasch Adding specs to guarantee that the Kafka client will be initialised with the expected configurations is quite complicated due to the fact that the dependency is hidden in the internals. I started mocking but the test became long and complicated.

I am thinking in a way that it could be added without many changes, but without having a public interface for accessing or just to initialising the kafka client without produce a message is not that trivial.

Reply:
I don't think you need to add a spec of this – it's just passing in a parameter. So long as you've checked it out locally.